python/doc/examples/reliability_sensitivity/sensitivity_analysis/plot_sensitivity_sobol_multivariate.py

```diff
@@ -0,0 +1,104 @@
+"""
+Estimate Sobol' indices for a function with multivariate output
+===============================================================
+"""
+# %% 
+#
+# In this example, we estimate the Sobol' indices of a function by sampling methods. 
+# This function has several outputs, which leads to the need of aggregated Sobol' indices. 
+#
+
+# %%
+# Introduction
+# ------------
+#
+# In this example we are quantify the sensitivity of inputs to outputs of a function based on Sobol' indices.
+#
+# The function we consider has 5 outputs. In this case, it may be convenient to consider each output separately. It may also be interesting to aggregate the sensitivity indices to that we get a global understanding of the sensitivity of the inputs to the average output. 
+
+# %%
+# Define the model
+# ----------------
+
+# %%
+import openturns as ot
+import openturns.viewer
+import openturns.viewer as viewer
+ot.Log.Show(ot.Log.NONE)
+
+# %%
+# We define a linear model with 5 independent Gaussian inputs and 2 ouputs.
+
+inputDistribution = ot.Normal(5)
+function = ot.SymbolicFunction(['x0', 'x1', 'x2', 'x3', 'x4'], 
+                               ['x0 + 4.0 * x1 ^ 2 + 3.0 * x2', 
+                                '-7.0 * x2 - 4.0 * x3 + x4'])
+
+# %%
+# Estimate the Sobol' indices
+# ---------------------------
+
+# %%
+# We first create a design of experiments with the `SobolIndicesExperiment`.
+
+# %%
+size = 1000
+sie = ot.SobolIndicesExperiment(inputDistribution, size)
+inputDesign = sie.generate()
+input_names = inputDistribution.getDescription()
+inputDesign.setDescription(input_names)
+print("Sample size: ", inputDesign.getSize())
+
+# %%
+# We see that 7000 function evaluations are required to estimate the first order and total Sobol' indices.
+# Then we evaluate the outputs corresponding to this design of experiments.
+
+# %%
+outputDesign = function(inputDesign)
+
+# %%
+# Then we estimate the Sobol' indices with the `SaltelliSensitivityAlgorithm`. 
```

Comment:
I think Saltelli is likely to disappear in a future version of OT. It might be better to showcase Martinez instead.

Reply:
Very true. Actually, I plan to perform a bench and show that this is the best. The theory is also better for this one: I have a presentation to explain this, perhaps a future cotech.